Cargo.toml

```diff
@@ -44,10 +44,11 @@ hex = { version = "0.4.3" , features = ["serde"] }
 prio = { path = ".", features = ["test-vector"] }
 
 [features]
-default = []
+default = ["internal_prg"]
 test-vector = ["rand", "serde_json"]
 multithreaded = ["rayon"]
 prio2 = ["aes-gcm", "ring"]
+internal_prg = ["aes", "ctr", "cmac"]
```

Comment:
Can we pick a more descriptive name than `internal_prg`?

Reply:
Do you have a suggestion?

Reply:
"internal_prg_aes128"? A comment would also be helpful here.
```
# This feature controls whether VDAFs that use `PrgAes128` will be compiled.
```

Reply:
In #267 we discussed the risk of combinatorial explosion of features. With that in mind, I don't want us to end up with one feature flag for every PRG implementation we end up with. It seems like what's at issue here is cryptography dependencies that Simon doesn't want. What if we name the feature `builtin_cryptography`? The idea being: if you turn on this feature (which is on by default), `prio` will make opinionated choices about crypto dependencies and use them. If you don't turn it on, you need to BYO crypto and adapt it to the cryptography traits that `prio` exports. Going forward, if and when we add new cryptography dependencies to `libprio-rs`, we can gate them behind this existing feature instead of having to add new ones.

Reply:
We did however also disable some "builtin_cryptography" with the prio2 feature. This is specifically about the PRG part.

Reply:
@simon-friedberger that's a somewhat separate issue since "prio2" hides the legacy stuff. I think @tgeoghegan's suggestion makes sense: A "catch-all" term is perhaps best. Perhaps "rust_crypto_deps", or just "crypto_deps"?